### ISSUE TITLE
Warn about multi-digits fds in ksh

### ISSUE THREAD

**Issue by koalaman** [koalaman/shellcheck#171]

**Comment by Gandalf-** [comment]
This has been fixed
```haskell
    bashism (T_FdRedirect id num _)
        | all isDigit num && length num > 1 = warnMsg id "FDs outside 0-9 are"
```
- https://github.com/koalaman/shellcheck/blob/master/src/ShellCheck/Checks/ShellSupport.hs#L195-L196
- https://github.com/koalaman/shellcheck/commit/08f7ff37c590f8aa6dba6d989d707aad4adc5f4c is as far back as git blame would take me

**Comment by koalaman** [comment]
This check is just for `sh` and `dash`. It doesn't trigger for `Ksh`:

    shellcheck -s ksh - <<< 'echo 42> file'
